mythril/ethereum/util.py

```diff
@@ -139,7 +139,11 @@ def solc_exists(version):
     return solc_binary
 
 
-all_versions = solcx.get_installable_solc_versions()
+try:
+    all_versions = solcx.get_installable_solc_versions()
+except:
```

Comment:
Adding Something like
```suggestion
except ConnectionError:
```
might be better.
PS: Don't forget to import it from `requests`